Makefile

```diff
@@ -30,6 +30,14 @@ test-e2e:
 	./openshift/e2e-tests.sh
 .PHONY: test-e2e
 
+perf-tests:
+	export ES_DEVELOPMENT=true && \
```

Comment:
nit, spacing is a bit off here. Not sure if this is a problem or not.

Reply:
This is weird because in my IDE it shows ok. I will check it.